_posts/2022-03-12-preregistration_of_qualitative_studies.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Preregistration of qualitative studies"
```

Comment:
I think we could rename it "preregistration of research protocols"?
As it is not only valid for qualitative studies - although we thought of including this mainly to address requests from qualitative researchers that feel like the other guidelines are mainly quantitative research focused.

Reply:
I think we can do either way, and you can decide which?